plugin-server/src/httpserver.ts

```diff
@@ -0,0 +1,31 @@
+import { createServer, IncomingMessage, ServerResponse } from 'http'
+
+import { healthcheck } from './healthcheck'
+import { Status } from './utils/status'
+
+const port = 5000
+
+const server = createServer(async (req: IncomingMessage, res: ServerResponse) => {
+    if (req.url === '/health' && req.method === 'GET') {
+        const status = await healthcheck()
```

Comment:
`healthcheck` here needs some refactoring/forking to make this actually useful. Look forward to that in the next PR!